src/components/fpl/LeagueTables.tsx

```
@@ -0,0 +1,809 @@
+              <div className="w-3 h-3 bg-green-500 rounded-full animate-pulse"></div>
+              <h4 className="text-lg font-semibold text-gray-900 dark:text-white">Live Tracking Controls</h4>
+            </div>
+            
+            {/* Controls Grid */}
+            <div className="grid grid-cols-1 lg:grid-cols-2 gap-6">
+              {/* League Selection */}
+              <div className="space-y-3">
+                <label className="block text-sm font-medium text-gray-700 dark:text-gray-300">
+                  📊 League Selection
+                </label>
+                <select
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)